test/test_other.py

```diff
@@ -15141,3 +15141,15 @@ def test_extra_struct_info(self):
   def test_fs_writev_partial_write(self):
     self.set_setting('FORCE_FILESYSTEM')
     self.do_run_in_out_file_test('fs/test_writev_partial_write.c')
+
+  @requires_v8
+  @parameterized({
+    '': [[]],
+    'O2': [['-O2']]
+  })
+  def test_fp16(self, opts):
+    self.v8_args += ['--experimental-wasm-fp16']
+    # TODO Remove this. Liftoff is currently broken for this test.
```

Comment:
Out of curiosity, do you know what specifically is broken?

Reply:
https://chromium-review.googlesource.com/c/v8/v8/+/5842546